### PR TITLE
Email preview

### PR DESCRIPTION
**Pull request by jorgehs91**
### Description (*)

1. Added a _module.less to the module to set the height of the email preview iframe.

### Reported Issue
https://github.com/magento/magento2/issues/23876

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by kalpmehta** [comment]
@jorgehs91 Hi, can you please add steps to reproduce and/or add the link to the reported issue? Thanks

**Comment by jorgehs91** [comment]
Hi @kalpmehta , i've edited my post with the link to the original issue.

**Review by kalpmehta — APPROVED** [review]

**Comment by kalpmehta** [comment]
@jorgehs91 Thanks for your contribution. Please sign the CLA otherwise we cannot accept your PR.

**Comment by jorgehs91** [comment]
@kalpmehta Thanks for the tips. The CLA has been accepted.

**Comment by jorgehs91** [comment]
Do I have to do something more? Or just wait it pass?

--Update
Nevermind, it has been approved.
Thanks.

**Comment by jorgehs91** [comment]
@melnikovi Hello, can you review this PR?
Do I need to do something else?

Thanks!

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed 

**Before:**
before

**After:**
after

File: app/code/Magento/Email/view/adminhtml/web/css/source/_module.less [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+//  */
+
+//
+//  Common
+//  _____________________________________________
+
+& when (@media-common =true) {
+    .cms-revision-preview {
+        iframe {
+            height: 98vh;
```
Comment (melnikovi) [inline app/code/Magento/Email/view/adminhtml/web/css/source/_module.less]: Thank you for your contribution. Could you investigate what caused the issue? Or it was always like this? Also need to add fullback for IE 11 or redo the fix.

**Review by jorgehs91 — COMMENTED** [review]
I have made another commit to fix the compatibility

**Review by melnikovi — APPROVED** [review]

**Review by sidolov — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sidolov, thank you for the review.
[ENGCOM-5848](https://jira.corp.magento.com/browse/ENGCOM-5848) has been created to process this Pull Request

**Comment by engcom-Alfa** [comment]
@magento give me test instance

**Comment by magento-engcom-team** [comment]
Hi @engcom-Alfa. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @engcom-Alfa, here is your [new Magento instance](https://pr-24599.instances.magento-community.engineering).
Admin access: [https://pr-24599.instances.magento-community.engineering/admin](https://pr-24599.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by engcom-Alfa** [comment]
Hi @jorgehs91 
We checked the PR with the last commit and looks like changes doesn't solve the problem.

**Actual Result:**
after2

Checked on Chrome, Firefox and Internet Explorer (the result is the same at all)

@jorgehs91 Could you take a look, please?

Thanks!

**Comment by sidolov** [comment]
@jorgehs91 , I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for the collaboration!

**Pull request closed without merging**